docs/db.md

```diff
@@ -0,0 +1,45 @@
+# Data Model
+
+## Tables
+
+### Users
+
+| Property | Type    | Notes                                                          |
+| -------- | ------- | -------------------------------------------------------------- |
+| id       | Integer | Auto-generated Primary Key for this row                        |
+| api_id   | String  | 26-character public-facing id made up of numbers and letters   |
+| email    | String  | Email address used to sign up for the service                  |
+| key      | Integer | ID of the public/private key set for this user. FK on Key Sets |
+
+### Items
+
+| Property | Type    | Notes                                                        |
+| -------- | ------- | ------------------------------------------------------------ |
+| id       | Integer | Auto-generated Primary Key for this row                      |
+| uid      | Integer | User that owns this item. Foreign Key on Users.              |
+| api_id   | String  | 26-character public-facing id made up of numbers and letters |
+| key      | Integer | Key used to decrypt this item. Foreign Key on Key Sets       |
+| overview | String  | Encrypted identifying info for the item                      |
+| details  | String  | Encrypted item details                                       |
+
+### Key Sets
+
+| Property    | Type    | Notes                                                      |
+| ----------- | ------- | ---------------------------------------------------------- |
+| id          | Integer | Auto-generated Primary Key for this row                    |
+| uid         | Integer | User that owns this key. Foreign Key on Users.             |
+| api_id      | String  | 26-character public-facing id made up of numbers & letters |
+| public_key  | String  | JWK with the unencrypted public key                        |
+| private_key | String  | Encrypted private key                                      |
+| account_key | String  | Key that unlocks the user's data                           |
+
+## Relationships Between Entities
```

Comment:
nit - should we also define in this doc what indexes we plan to define?

Reply:
@carlosmmc PostgreSQL creates indexes on unique columns [automatically](https://www.postgresql.org/docs/current/indexes-unique.html), and that should be all we need.